docs/content.zh/docs/deployment/config.md

```diff
@@ -470,7 +470,7 @@ using State Changelog.
 
 #### FileSystem-based Changelog options
 
-These settings take effect when the `state.backend.changelog.storage`  is set to `filesystem` (see [above](#state-backend-changelog-storage)).
```

Comment:
Thanks for the pr.
I found the section is missed and committed a pr to fix it: 
https://github.com/apache/flink/pull/24354
https://github.com/apache/flink/pull/24355
Big thanks if you could help to review this firstly.
And the title should also be modified after above pr is merged.

Reply:
Done & force pushed. Thanks!